app/lib/shared/markdown.dart

```diff
@@ -8,8 +8,10 @@ import 'package:path/path.dart' as p;
 
 final Logger _logger = new Logger('pub.markdown');
 
-final List<m.BlockSyntax> _blockSyntaxes =
```

Comment:
Could/should we just use the GitHub extension set directly – then add the Header/Setext things?

Reply:
We do remove the `InlineHtmlSyntax` from the github extension set. Not sure the reason, maybe to prevent HTML injection?